tox.ini

```diff
@@ -34,5 +34,7 @@ skip_install = true
 [testenv:mypy]
 basepython = python3.8
 deps = mypy
-commands = mypy {posargs} src
+extras = test
+    trio
```

Comment:
Should this rather be the trio stubs that are distributed separately?

Reply:
Possibly, but that results in more fun type errors that's coming in a new pr

Reply:
If trio itself does not carry type annotations, what is the point of adding trio as a dependency here?